deploy/kubernetes/base/controller/kustomization.yaml

```diff
@@ -5,5 +5,4 @@ namespace:
 resources:
 - cluster_setup.yaml
 - controller.yaml
-- psp.yaml
```

Comment:
this change in base also means stable-1.23, stable-1.22, stable-1.21 wont have the psp. in this PR we are adding psp to the 1.24 overlay. should we consistent (i.e either remove psp from all 1.21 and beyond or keep psp in 1.21, 1.22, 1.23, 1.24 and removed from 1.25) ?

Reply:
Arg, you're absolutely right, good catch.

I think we should add it to the old ones so that stable-master is most simple.